features/build/stibuild.feature

```diff
@@ -94,8 +95,9 @@ Feature: stibuild.feature
     When a pod becomes ready with labels:
       | deploymentconfig=git |
       | deployment=git-2     |
+    Given I obtain test data file "build/httpd-ex.git"
```

Comment:
testdata/build/httpd-ex.git is a dir,then I met this error
    `Given I obtain test data file "build/httpd-ex.git" # features/step_definitions/file.rb:1`
      `Is a directory - read (Errno::EISDIR)`

Could I change this back?

Reply:
Let's create a step `I obtain test data dir ...`. Do you think you can copy existing step and make it work with a directory? Or do you want dx team do it?

Reply:
@xiuwang ^^

Reply:
@akostadinov I will do that, thanks for advice